apps/web/src/modules/job/components/card/index.tsx

```diff
@@ -1,51 +1,69 @@
 import { Divider, Typography } from '@mui/material'
 import React from 'react'
 
-import Footer from './footer'
-import Header from './header'
-import ShootingDetail from './shootingDetail'
+import Footer from './components/footer'
+import Header from './components/header'
+import ShootingDetail from './components/shootingDetail'
```

Comment:
I forget this in the last review. Component folder name should also be PascalCase (ex. `/components/ShootingDetail`).